pkgs/tools/networking/maubot/default.nix

```diff
@@ -0,0 +1,72 @@
+{ lib
+, python3
+}:
+let python = python3.override {
```

Comment:
```suggestion
{ lib, python3 }:

let python = python3.override {
```

Reply:
One argument per line makes for nicer diffs, so I will mark this suggestion as resolved.

Reply:
Fair, it's something reviewers can't ever agree on :laughing:.

In general I wouldn't have made this suggestion if there had been more arguments, with only 2, things never become hard to look at in diffs imo.

It would be great if we could still get a new line after the arguments, separating it from the overlays.